src/components/author/author.twig

```diff
@@ -1,23 +1,76 @@
+{#
+  This component accepts the following properties
+  - `meta` (optional, string)
+  - `date` (optional, date object)
+
+  If both the `meta` and `date` properties contain values, the `meta` value has
+  a higher priority and will override the `date` value.
+
+  It is recommened to use the `date` property when wanted to render a date as the
+  meta data so that a more a accessible user experience can be rendered.
+#}
+{% set _meta_content %}
+  {% block meta_content %}
+    {% if meta %}
+      <p>{{ meta }}</p>
+    {% elseif date %}
+      <p>
+        {# This creates a more accessible, and less confusing, user experience #}
+        <span class="u-hidden-visually">
+          Published on {{ date|date('F jS, Y') }}
+        </span>
+        {#
+          The <time> element is not well supported by screen readers, therefore,
+          it is aria-hidden. On macOS VoiceOver, for example, the date would be
+          read twice, which was a confusing experience.
+
+          @see: https://twitter.com/LeonieWatson/status/1333078194925264898
+
+          The `<time>` element was not removed because per MDN Web Docs,
+          it allows for better search engine results:
+
+          > The `<time>` HTML element represents a specific period in time.
+          It may include the `datetime` attribute to translate dates into
+          machine-readable format, allowing for better search engine results or
+          custom features such as reminders.
+
+          @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time
+        #}
+        <time aria-hidden="true" datetime="{{ date|date('Y-m-d') }}">
+          {{ date|date('F jS, Y') }}
+        </time>
+      </p>
+    {% endif %}
+  {% endblock %}
+{% endset %}
+
 <div class="c-author o-media{% if class %} {{ class }}{% endif %}">
+  {# Avatar(s) #}
   {% embed '@cloudfour/objects/bunch/bunch.twig' with { class: 'o-media__object' } %}
     {% block content %}
       {% for author in authors %}
         {% include '@cloudfour/components/avatar/avatar.twig' with { src: author.avatar } only %}
       {% endfor %}
     {% endblock %}
   {% endembed %}
-  <div class="o-media__content">
-    <div>
-      <span class="u-hidden-visually">Authored by</span>
+
+  {# Author content #}
+  <div class="c-author__content-container o-media__content">
+    {# Author links #}
+    <p class="c-author__author">
+      <span class="u-hidden-visually">By</span>
```

Comment:
In the spirit of your date prefix suggestion, should this also be configurable? (But I still think this is a good default!)

Reply:
Updated! https://github.com/cloudfour/cloudfour.com-patterns/pull/1607/commits/f4074b232547985fbcecfeb2f456ce72ce0baf40